test/Atmos/Parameterizations/Microphysics/runtests.jl

```diff
@@ -0,0 +1,139 @@
+using Test
+using CLIMA.Microphysics
+using CLIMA.MicrophysicsParameters
+using CLIMA.MoistThermodynamics
+
+@testset "RainDropFallSpeed" begin
+  # two typical rain drop sizes
+  r_small = 0.5 * 1e-3
+  r_big   = 3.5 * 1e-3
+
+  # example atmospheric conditions
+  p_range = [1013., 900., 800., 700., 600., 500.] .* 100
+  T_range = [  20.,  20.,  15.,  10.,   0., -10.] .+ 273.15
+  ρ_range = p_range ./ R_d ./ T_range
+
+  # previousely calculated terminal velocity values
+  ref_term_vel_small = [4.44,  4.71,  4.96, 5.25, 5.57, 5.99]
+  ref_term_vel_big   = [11.75, 12.47, 13.11, 13.90, 14.74, 15.85]
+
+  for idx in range(Int(1), stop=Int(6))
+    vc = terminal_velocity_single_drop_coeff(ρ_range[idx])
+
+    term_vel_small = vc .* sqrt(r_small .* grav)
+    term_vel_big   = vc .* sqrt(r_big   .* grav)
+
+    @test term_vel_small ≈ ref_term_vel_small[idx] atol = 0.01
+    @test term_vel_big   ≈ ref_term_vel_big[idx]   atol = 0.01
```

Comment:
These tests are fine, but seem brittle: if we change formulation of the terminal velocity (e.g., by changing size distribution of raindrops), they will fall. Wouldn't it be better to test universal limits? (e.g., density liquid = density air, density liquid >> density air, 4xg leads to 2x terminal velocity etc.)

Reply:
Those would be good tests. But I don't know how to overwrite the dens_liquid and grav, because they are planet parameters. I'll add an issue to improve on the tests for Microphysics

Reply:
issue #280